content/reference/current/plugins/azure_devops/index.md

```diff
@@ -0,0 +1,106 @@
+{{< /alert >}}
+
+## Requirements
+
+You will need:
+
+- A git repository: we will be using GitHub in our example
+- A working simulation on a [Gatling Enterprise Self-Hosted](https://gatling.io/enterprise/) instance: you will need its ID later on, which you can copy with **Copy Simulation ID to clipboard** in the Simulations page
+- An [API Token]({{< ref "../../admin/api-tokens" >}}) with the permissions to configure the previous simulation
+
+On Azure DevOps, your will need:
+
+- An Azure DevOps project
+- An agent pool:
+  - You can learn how to create one by following the official [Azure Devops documention](https://learn.microsoft.com/en-us/azure/devops/pipelines/agents/agents)
+  - You will need the agent pool's name later on, in our example we will use `AzurePipelinesAgentsPool`
+
+## Preparing the repository
+
+First, [download the CI script](https://downloads.gatling.io/releases/frontline-ci-script/{{< var ciPluginsVersion >}}/frontline-ci-script-{{< var ciPluginsVersion >}}.zip), unzip it and put the content at the root of the repository. Make sure it is named `start_simulation.sh`.
+
+{{< alert info >}}
+If you want to learn more about this CI script, you can check its [documentation page]({{< ref "../script/#using-a-shell-script" >}}).
+{{< /alert >}}
+
+Create a file called `azure-pipelines.yml` at the root of the repository with the following content:
+
+```yaml
+trigger: none
+
+pool: AzurePipelinesPool # Name of the agent pool configured in Azure DevOps
+
+steps:
+- task: Bash@3
+  displayName: Install JQ
+  inputs:
+    targetType: 'inline'
+    # Curl and JQ are required by the CI script
+    # In this example, we use Ubuntu agents which come with curl but not jq
+    script: 'sudo apt-get update && sudo apt-get -y install jq'
+- task: Bash@3
+  displayName: Start simulation
+  inputs:
+    filePath: 'start_simulation.sh'
+    arguments: >
+      $(gatlingEnterpriseUrl)
+      $(apiToken)
+      $(simulationId)
+```
+
+{{< alert info >}}
+The CI script requires both Curl and JQ.
+Make sure they are already installed on your agents or install them in a preliminary step like we did above for JQ.
+{{< /alert >}}
+
+## Creating a pipeline
+
+On Azure DevOps, click on **Pipelines** in the left menu, then on **Create Pipeline** or **New Pipeline**.
+
+From here, you will follow a 4 steps process: Connect -> Select -> Configure -> Review.
+
+1. On the **Connect** step, select the type of the repository you want to use. We will be using GitHub.
+2. On the **Select** step, select the repository your created earlier.
+
+If you never connected Azure DevOps to your GitHub account, you'll be prompted by GitHub to authorize the Azure DevOps application to access your repositories on your behalf.
+
+After your account is linked, you will be able to browse your repositories. When using an organization repository, click on **All repositories** on the top right.
+
```

Comment:
To avoid start again for "3."
```suggestion
2. On the **Select** step, select the repository your created earlier.

    If you never connected Azure DevOps to your GitHub account, you'll be prompted by GitHub to authorize the Azure DevOps application to access your repositories on your behalf.

    After your account is linked, you will be able to browse your repositories. When using an organization repository, click on **All repositories** on the top right.

```